src/neo/SmartContract/Native/Oracle/OracleContract.cs

```diff
@@ -0,0 +1,231 @@
+using Neo.Cryptography.ECC;
+using Neo.IO;
+using Neo.Ledger;
+using Neo.Persistence;
+using Neo.SmartContract.Manifest;
+using Neo.VM;
+using Neo.VM.Types;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Numerics;
+using System.Text;
+using Array = Neo.VM.Types.Array;
+
+namespace Neo.SmartContract.Native.Oracle
+{
+    public sealed class OracleContract : NativeContract
+    {
+        public override string ServiceName => "Neo.Native.Oracle";
+
+        public override int Id => -4;
+
+        private const byte Prefix_Validator = 24;
+        private const byte Prefix_Config = 25;
+        private const byte Prefix_PerRequestFee = 26;
+
+        public OracleContract()
+        {
+            Manifest.Features = ContractFeatures.HasStorage;
+        }
+
+        internal override bool Initialize(ApplicationEngine engine)
+        {
+            if (!base.Initialize(engine)) return false;
```

Comment:
We should check the storage in order to avoid duplicate calls

Reply:
Done

Reply:
@doubiliu Use `Storage.TryGet` to fix this issue.